invesalius/gui/task_navigator.py

```diff
@@ -305,8 +301,8 @@ def __init__(self, parent):
         self.ref_mode_id = const.DEFAULT_REF_MODE
 
         # Initialize list of buttons and numctrls for wx objects
-        self.btns_coord = [None] * 7
-        self.numctrls_coord = [list(), list(), list(), list(), list(), list(), list()]
+        self.btns_coord = [None] * 6
```

Comment:
Prefer this way:
`self.btns_coord = [None, None, None, None, None, None]`
`self.numctrls_coord = [[], [], [], [], []]`